controls/raddiagram/extensions/settingspane/settingspane_overview.md

```diff
@@ -1,143 +1,143 @@
+
+In most __Diagramming__ examples you will find the __SettingsPane__ applied as an __AdditionalContent__ on the diagramming surface. This way the control is displayed next to a focused __RadDiagramItem__ thus allowing users to dynamically change the look and feel of the item.  In order to display the __SettingsPane__ this way in your application as well, you can add it through the __ItemInformationAdorner.AdditionalContent__ attached property as demonstrated in __Example 1__. Where the primitives namespace is defined like this:	
+
+>tip xmlns:primitives="clr-namespace:Telerik.Windows.Controls.Diagrams.Primitives;assembly=Telerik.Windows.Controls.Diagrams"
+
+#### __[XAML] Example 1: Adding SettingsPane in XAML__
+{{region raddiagram-extensions-settingspane-0}}
+	<telerik:RadDiagram x:Name="diagram">
+	    <primitives:ItemInformationAdorner.AdditionalContent>
+	        <telerik:SettingsPane Diagram="{Binding ElementName=diagram}" />
+	    </primitives:ItemInformationAdorner.AdditionalContent>
+	</telerik:RadDiagram>
+{{endregion}}
+    
+![Rad Diagram Settings Pane Overview](images/RadDiagram_SettingsPane_Overview.png)
+
+## Customization
+
+The __SettingsPane__ has a default view that can be used out-of-the-box in scenarios where you only need to display the common settings of a __RadDiagramItem__. However you can also customize the content of the pane to represent more specific information. These are the basic tasks you might need to implement while customizing your __SettingsPane__ instance:		
+
+* __Change the tab headers__ - in order to change the headers of the tabs in the default __SettingsPane__, you can change the value of their localization strings. Please take a look at the [Localization]({%slug raddiagram-localization%}) article to find the localization string of each tab displayed inside the __SettingsPane__.			
+
+* __Add and remove tabs__ - in order to add or remove tabs from the default __SettingsPane__, you need to edit the __ControlTemplate__ of the __SettingsPaneView__ as this is the control that represents the content of the __SettingsPane__. Please take a look at the [SettingsPaneView ControlTemplate Structure](#settingspaneview-controltemplate-structure) section for more information.			
+
+* __Edit the content of an existing tab__ - The content of each of the four default __SettingsPane__ tabs – Home, Size, Style, Text, is represented by a different control:			
+	* __SettingsPaneHomeControl__ - representing the content of the __Home__ tab
+	
+	* __SettingsPaneSizeControl__ - representing the content of the __Size__ tab				
+
+	* __SettingsPaneStyleControl__ - representing the content of the __Style__ tab
+
+	* __SettingsPaneTextControl__ - representing the content of the __Text__ tab Furthermore, the content of each control is described in its __ControlTemplate__. This is why in order to change the existing content of a __SettingsPane__ tab, you need to edit the __ControlTemplate__ of the appropriate control.
```

Comment:
Not sure what "content of each control is described in its __ControlTemplate_" means. I think we can just say "In order to change the existing content of a __SettingsPane__ tab, you need to edit the __ControlTemplate__ of the appropriate control."